setup.py

```diff
@@ -19,7 +19,7 @@
 URL = "https://github.com/ConsenSys/mythril"
 AUTHOR = "ConsenSys Dilligence"
 AUTHOR_MAIL = None
-REQUIRES_PYTHON = ">=3.5.0"
+REQUIRES_PYTHON = ">=3.6.0"
```

Comment:
Is this installed by default in Ubuntu now? / Do we have info on the environments of the people that use mythril?

Reply:
People who log issues mainly use docker. So it should be fine. The latest Ubuntu has 3.7.5, only the ancient ones like Ubuntu 16(which is dead) has python 3.5 installed by default.